### ISSUE TITLE
Npm install

### ISSUE THREAD

**Issue by mkerfoot** [evancohen/smart-mirror#31]
I'm getting the following error after performing the following cmds. 

CD /*/smart-mirror
Npm install

Bash: npm: command not found

**Comment by monaiz** [comment]
Are you doing it from the root?

**Comment by mkerfoot** [comment]
I am.

Root@raspberrypi:/boot/smart-mirror#

I ended up running sudo apt-get install nodejs npm and then I could run the cmd install npm but with a lot of errors. Then I tried running npm start and got errors(as expected since the npm install errored out so much). 


Error

Command "usr/bin/nodejs" "/usr/bin/npm" install
cwd /boot/smart-mirror#

Thanks for any help, I'd love to get this working.

Wait is root/administrator different than what sudo su brings me to? 

Sorry I'm a noob
